tailcfg/tailcfg.go

```diff
@@ -343,6 +371,16 @@ type Node struct {
 	ExitNodeDNSResolvers []*dnstype.Resolver `json:",omitempty"`
 }
 
+// HasCap reports whether the node has the given capability.
+func (v NodeView) HasCap(cap NodeCapability) bool {
+	return v.ж.HasCap(cap)
+}
+
+// HasCap reports whether the node has the given capability.
+func (v *Node) HasCap(cap NodeCapability) bool {
```

Comment:
HasCap here vs HasCapability on the other type is weird. pick a name?

Reply:
Alternative, HasCap here but just "Contains" on the other one, since it's only about capabilities it doesn't need "Cap" or "Capabilities" in its name and "Contains" seems like the common name.